packages/api/src/listeners/PluginServerRequest.ts

```diff
@@ -1,30 +1,62 @@
 import { Listener } from '@sapphire/framework';
 import type { ApiRequest } from '../lib/structures/api/ApiRequest';
 import type { ApiResponse } from '../lib/structures/api/ApiResponse';
-import { ServerEvents } from '../lib/structures/http/Server';
+import type { MethodName } from '../lib/structures/http/HttpMethods';
+import { ServerEvent } from '../lib/structures/http/Server';
```

Comment:
Better to name this PluginApiEvents I think

Reply:
I personally prefer the current name, it doesn’t conflict with Sapphire’s or Discord.js’s enum